frostsnap_core/src/appkey.rs

```diff
@@ -0,0 +1,44 @@
+use crate::tweak::{TweakableKey, Xpub};
+use schnorr_fun::fun::prelude::*;
+
+/// A 65-byte encoded point and chaincode. This is exists because it's easier to pass around byte
+/// arrays via FFI rather than `Point`.
+#[derive(Clone, Copy, PartialEq, Eq, Ord, PartialOrd, Hash)]
+pub struct Appkey(pub [u8; 65]);
```

Comment:
nit: could rename Appkey -> AppKey (not fussed)

Reply:
I made it `appkey` and `rootkey` everywhere. They are fixed concepts and use frequently so simplifying the name helps.